jdaviz/configs/default/tests/test_data_menu.py

```diff
@@ -146,6 +147,35 @@ def test_data_menu_remove_subset(specviz_helper, spectrum1d):
     # assert dm.layer.choices == ['test', 'test2', 'Subset 2']
 
 
+def test_data_menu_dq_layers(imviz_helper):
+    imviz_helper.load_data(np.zeros((2, 2)), data_label='image', show_in_viewer=True)
+
+    hdu = fits.ImageHDU(np.zeros(shape=(2, 2), dtype=np.int32))
+    hdu.name = 'DQ'
+    dq = fits.HDUList([fits.PrimaryHDU(), hdu])
+
+    imviz_helper.load_data(dq, data_label="dq", show_in_viewer=True)
+    # manually link this DQ to the parent image
+    imviz_helper.app._set_assoc_data_as_child("dq[DQ,1]", new_parent_label='image')
```

Comment:
would be nice for this to be done automatically or a way from `load_data` (there currently is from `add_data` internally).  Or maybe users would never be expected to load DQ data themselves and we want to keep this internal to the parser and DQ plugin? 🤷‍♂️

Reply:
I'm not sure we currently support that without internal calls.  @bmorris3?

Reply:
(The implementation should work for that case though, I just don't want to blow up the scope to something that technically isn't implemented elsewhere)

Reply:
if you create a FITS HDUList containing the data and the DQ entries, and load them with `ext=(`SCI`, `DQ`)` you should get that behavior without the internal methods.

Reply:
true, but would be difficult to attach to an existing spectrum1d cube or something like that.  Ultimately I don't think this is a use-case users need (yet), and so this probably does the trick for test coverage just fine 🤷‍♂️